pallets/attestation/src/lib.rs

```diff
@@ -210,10 +210,7 @@ pub mod pallet {
 		AttestationRevoked(AttesterOf<T>, ClaimHashOf<T>),
 		/// An attestation has been removed.
 		/// \[account id, claim hash\]
-		AttestationRemoved(AttesterOf<T>, ClaimHashOf<T>),
-		/// The deposit owner reclaimed a deposit by removing an attestation.
-		/// \[account id, claim hash\]
-		DepositReclaimed(AccountIdOf<T>, ClaimHashOf<T>),
+		AttestationRemoved(ClaimHashOf<T>),
```

Comment:
I think it would be good to keep the attester DID, so that an attester could index events by that instead of keeping track of all the claim hashes it attested. Do you think it makes sense?

Reply:
So currently the Attester doesn't need to be the `attestation.owner` but could be anyone who is authorized to revoke this attestation. So who issues and who revoked an attestation are not necessarily the same.

I would also keep it and add an enum (like I explained in the description).

Reply:
I actually think that the information about the original attester is more useful than the information about who removed the attestation. Since you might use that information in a potential indexer, and not so much about who's actually removed it.

Reply:
But I see what you're saying. So your suggestion makes sense.

Reply:
Sorry just going over this once again: I don't think having the deposit reclaimer in the event is any useful. Rather, I would include the original (delegated) attester (the one that created the attestation), since that info is always available whenever we remove an attestation, even if by claiming its deposit back.

Reply:
So there are multiple possible entities at play here.

During the creation there is the Delegated attester and the real attester and during deletion, revoking it's similar. So there are potentially 4 different DIDs involved in the lifecycle of an attestation. The DID that issues the attestation, the Attester DID, the DID that revoked and the DID that deleted. The only DID that is easily queried is the attester DID since this is part of the attestation. The other three are only known if you check the delegation entry (which would require knowledge of all delegation methods and might get out of hand quite quick or might not even be possible at all) or if you search for the transaction and check the origin. I would therefore rather put the DID in there that did authorize the call and not the attester DID. Or as a middle ground maybe both?

Reply:
Yes, my point was that if I had to choose, I would rather choose the attester DID rather than the tx origin. I still don't see the value of showing the DID origin?

Reply:
The attester is easy to query. The DID that issued, removed, revoked the attestation is not easy to query. Making them part of the event would make it easy to get that information.

Reply:
I just noticed that at least for issuing, the attester DID is actually the origin of the transaction and the authorizationId is the organization where they belong to. So for issuing it's easy. For revocations and deletions it's a little more involved.